tests/src/main/java/org/hibernate/beanvalidation/tck/tests/constraints/constraintcomposition/ConstraintCompositionTest.java

```diff
@@ -121,8 +123,7 @@ public void testComposedConstraintsAreRecursive() {
 
 	@Test
 	@SpecAssertions({
-			@SpecAssertion(section = "3.3", id = "b"),
-			@SpecAssertion(section = "3.4", id = "s")
```

Comment:
Because it does not make sense with what was 3.4. s and I couldn't find an assertion making sense in 3.4.